Makefile

```diff
@@ -38,23 +37,14 @@ install_gems: check_bundle
 	fi
 
 install_pods: check_gems
-	@$(bundle_cmd) exec pod install --repo-update; \
+	@$(bundle_cmd) exec pod install; \
 	if [ $$? -eq 0 ]; then \
 		echo "All pods installed."; \
 	else \
 		echo "Error installing."; \
 		exit 1; \
 	fi
 
-update_pods: check_gems
-	@$(bundle_cmd) exec pod update; \
```

Comment:
I know this was added in a separate PR, but I don't think we'll ever want to run `pod update`. It'll almost always be `pod install`, and rarely `pod update <podname>`

Reply:
Do you want me to remove it?